server/middlewares/passport/passportAuthentication.js

```diff
@@ -0,0 +1,14 @@
+import passport from 'passport';
```

Comment:
I guess this file name too should be authentication since it is inside passport folder already